cloud_service/getting-started-rhacs-cloud-ocp.adoc

```diff
@@ -0,0 +1,104 @@
+. In the {cloud-console}, xref:../cloud_service/installing_cloud_other/cloud-create-instance-other.adoc#cloud-create-instance-other[create an *ACS Instance*].
+. In the ACS Console, xref:../cloud_service/installing_cloud_other/init-bundle-cloud-other-generate.adoc#init-bundle-cloud-other-generate[create an init bundle]. The init bundle contains secrets that allow communication between {product-title-managed-short} secured clusters and the ACS Console.
+. On each Kubernetes cluster, xref:../cloud_service/installing_cloud_other/init-bundle-cloud-other-apply#init-bundle-cloud-other-apply[apply the init bundle] by using it to create resources.
+. On each Kubernetes cluster, xref:../cloud_service/installing_cloud_other/install-secured-cluster-cloud-other.adoc#install-secured-cluster-cloud-other[install secured cluster resources] by using Helm charts or the `roxctl` CLI.
+. xref:../cloud_service/installing_cloud_other/verify-installation-cloud-other.adoc#verify-installation-cloud-other[Verify installation] by ensuring that your secured clusters can communicate with the ACS instance.
+
+[id="default-access-acs-console"]
+== Default access to the ACS Console
+
+By default, the authentication mechanism available to users is authentication by using Red Hat Single Sign-On (SSO).
+You cannot delete or change the Red Hat SSO authentication provider.
+However, you can change the minimum access role and add additional rules, or add another identity provider.
+
+[NOTE]
+====
+To learn how authentication providers work in ACS, visit xref:../operating/manage-user-access/understanding-authentication-providers.adoc[Understanding authentication providers].
+====
+
+A dedicated OIDC client of `sso.redhat.com` is created for each ACS Console. All OIDC clients share the same `sso.redhat.com` realm.
+Claims from the token issued by `sso.redhat.com` are mapped to an ACS-issued token as follows:
+
+* `realm_access.roles` to `groups`
+* `org_id` to `rh_org_id`
+* `is_org_admin` to `rh_is_org_admin`
+* `sub` to `userid`
+
+The built-in Red Hat SSO authentication provider has the required attribute `rh_org_id` set to the organization ID assigned to account of the user who created the {product-title-managed-short} instance.
+This is the ID of the organizational account the user is a part of. This can be thought of as the "tenant" the user is under and owned by.
+Only users with the same organizational account can access the ACS console by using the Red Hat SSO authentication provider.
+
+[NOTE]
+====
+To gain more control over access to your ACS Console, configure another identity provider instead of relying on the Red Hat SSO authentication provider.
+You can set up your own identity provider after provisioning by using one of the guides under *Operating -> Managing user access*.
+====
+[NOTE]
+====
+To configure this authentication provider to be the first authentication option on the login page, its name should be lexicographically smaller than `Red Hat SSO`.
```

Comment:
```suggestion
To configure this authentication provider to be the first authentication option on the login page, its name should be lexicographically smaller than the `Red Hat SSO`.
```

Reply:
I'm going to check with Eng but leave this as is for now.